### PR TITLE
Wiki lab fixes

### PR DESCRIPTION
**Pull request by unaihuete93**
# Module: 3
## Lab/Demo: Sharing Team Knowledge using Azure Project Wikis

Fixes # .

Changes proposed in this pull request:

- missing steps and clarifications 
-
-

### CODE REVIEW

File: Instructions/Labs/AZ400_M03_Sharing_Team_Knowledge_using_Azure_Project_Wikis.md [hunk 1]
```diff
@@ -79,8 +79,8 @@ In this task, you will publish a branch of an Azure DevOps repo as wiki.
...
-1.  In the vertical menu on the left side, click **Repos**, in the upper section of the **Files** pane, in the dropdown list, select **main**, and review the content of the main branch.
-1.  To the left of the **Files** pane, in the listing of the repo folder and file hierarchy, expand the **Documents** folder and its **Images** subfolder, in the **Images** subfolder, locate the **Website.png** entry, hover with the mouse pointer over its right end to reveal the vertical ellipsis symbol representing the **More** menu, click **More**, and, in the dropdown menu, click **Download** to download the **Website.png** file to the local **Downloads** folder on your lab computer. 
+1.  In the vertical menu on the left side, click **Repos**, in the upper section of the **Files** pane, make sure you have  the **TailwindTraders-Website** repo selected (choose it from the dropdown on the top with git icon). In the branch dropdown list (on top of "Files" with branch icon), select **main**, and review the content of the main branch.
```
Comment (mkennten) [inline Instructions/Labs/AZ400_M03_Sharing_Team_Knowledge_using_Azure_Project_Wikis.md:82]: @unaihuete93 Can you please change "git icon" to "Git icon"

File: Instructions/Labs/AZ400_M03_Sharing_Team_Knowledge_using_Azure_Project_Wikis.md [hunk 2]
```diff
@@ -188,7 +189,7 @@ In this task, you will manage the newly created project wiki.
 
 >**Note**: You will start by reverting the most recent change to the wiki page.
 
-1.  On you lab computer, in the Azure DevOps portal displaying the Wiki pane of the **Sharing Team Knowledge using Azure Project Wikis** project, with the content of the **Project Design** wiki selected, in the upper right corner, click the vertical ellipsis symbol and, in the dropdown menu, click view revisions.
+1.  On you lab computer, in the Azure DevOps portal displaying the Wiki pane of the **Sharing Team Knowledge using Azure Project Wikis** project, with the content of the **Project Design** wiki selected, in the upper right corner, click the vertical ellipsis symbol and, in the dropdown menu, click **View revisions**.
```
Comment (mkennten) [inline Instructions/Labs/AZ400_M03_Sharing_Team_Knowledge_using_Azure_Project_Wikis.md:192]: Can you please change "On you lab computer" to "On your lab computer". Was there before but then we get it fixed directly.

**Review by mkennten — CHANGES_REQUESTED** [review]
@unaihuete93 Looks good so far, just some minor changes need to be done. Thank you!

File: Instructions/Labs/AZ400_M03_Sharing_Team_Knowledge_using_Azure_Project_Wikis.md (hunk 2 above)
Comment (unaihuete93) [reply Instructions/Labs/AZ400_M03_Sharing_Team_Knowledge_using_Azure_Project_Wikis.md:192]: fixed :)

File: Instructions/Labs/AZ400_M03_Sharing_Team_Knowledge_using_Azure_Project_Wikis.md (hunk 1 above)
Comment (unaihuete93) [reply Instructions/Labs/AZ400_M03_Sharing_Team_Knowledge_using_Azure_Project_Wikis.md]: fixed! :)

**Comment by unaihuete93** [comment]
typos fixed :)

**Review by mkennten — APPROVED** [review]

**Pull request merged**